tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -1,14 +1,18 @@
 ---
 - name: Create hosted engine local vm
   block:
+  - set_fact: ansible_host_key_checking="{{ lookup('env','ANSIBLE_HOST_KEY_CHECKING') or true }}"
```

Comment:
Please notice that according to https://docs.ansible.com/ansible/latest/reference_appendices/config.html#host-key-checking
the user can also set host_key_checking in one of the ansible.cfg files and in that case this will not detect it.

Reply:
Maybe this: https://docs.ansible.com/ansible/latest/plugins/lookup/config.html